src/commands/lambda/__tests__/functions/instrument.test.ts

```diff
@@ -458,7 +458,181 @@ describe('instrument', () => {
         '`apiKeySecretArn` is not supported for Node runtimes when using Synchronous Metrics. Use either `apiKey` or `apiKmsKey`.'
       )
     })
+
+    describe('test the new universal instrumentation workflow for Java and .Net', () => {
+      const region = 'us-east-1'
+      const config = {
+        FunctionArn: 'arn:aws:lambda:us-east-1:123456789012:function:lambda-hello-world',
+        Handler: 'index.handler',
+        Layers: [],
+        Runtime: 'runtime',
+      }
+      const settings = {
+        flushMetricsToLogs: false,
+        layerAWSAccount: mockAwsAccount,
+        layerVersion: 13,
+        mergeXrayTraces: false,
+        tracingEnabled: false,
+      }
+      const dotnetRuntime = 'dotnet6'
+      const javaRuntime = 'java11'
+
+      describe('test for runtime ${dotnetRuntime}', () => {
```

Comment:
I think you have to use backticks for template strings to work in javascript
```suggestion
      describe(`test for runtime ${dotnetRuntime}`, () => {
```